File: app/src/main/java/com/example/friendlocation/friendlocation/JavaClasses/Model/Friend.java
package com.example.friendlocation.friendlocation.JavaClasses.Model;

import android.graphics.drawable.Drawable;
import android.widget.ImageView;

/**
 * Created by barte_000 on 19.03.2017.
 */

public class Friend {
    String userId;
    String name;
    Drawable avatar;

    public Friend(String userId, String name, Drawable avatar) {
        this.userId = userId;
        this.name = name;
        this.avatar = avatar;
    }

    public Friend(String userId, String name) {
        this.userId = userId;
        this.name = name;
    }

    public Drawable getAvatar() {
        return avatar;
    }

    public void setAvatar(Drawable avatar) {
        this.avatar = avatar;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }


    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }


}

File: app/src/main/java/com/example/friendlocation/friendlocation/JavaClasses/Model/Friend.kt
package com.example.friendlocation.friendlocation.JavaClasses.Model

import android.graphics.drawable.Drawable
import android.widget.ImageView

/**
 * Created by barte_000 on 19.03.2017.
 */

class Friend {
    var userId: String
    var name: String
    lateinit var avatar: Drawable

    constructor(userId: String, name: String, avatar: Drawable) {
        this.userId = userId
        this.name = name
        this.avatar = avatar
    }

    constructor(userId: String, name: String) {
        this.userId = userId
        this.name = name
    }


}
